src/core/main.c

```diff
@@ -1998,6 +1998,8 @@ static int invoke_main_loop(
                                 /* Reloading failed before the point of no return.
                                  * Let's continue running as if nothing happened. */
                                 m->objective = MANAGER_OK;
+                        else
+                                log_info("Reloading finished.");
```

Comment:
Hmm, shouldn't we print the time diff here then? I know it can be calculated, but it's probably some fraction of a second and it'd be nice to say how much it took.

Now that we have a "start" message and a "finish" message, maybe change the first one to "Reloading..." ?

Reply:
done